marketplace-ui/src/app/modules/product/product-detail/product-detail.component.ts

```diff
@@ -372,7 +374,13 @@ export class ProductDetailComponent {
     return tag;
   }
 
+  updateTitle() {
+    const title = this.productDetail().names[this.languageService.selectedLanguage()];
```

Comment:
I got error in console tab here 

![image](https://github.com/user-attachments/assets/fb39b5b1-505b-4159-abf9-496d5abb122f)